Makefile

```diff
@@ -96,25 +96,35 @@ help: ## Show this help screen
 .PHONY: all
 all: manager ## Build all binaries
 
-$(GO-LICENSES):
+$(TIMESTAMPS_DIR)/go-licenses-installed:
 	go install github.com/google/go-licenses@latest
+	@mkdir -p $(TIMESTAMPS_DIR) && touch $@
+
+go-licenses: $(TIMESTAMPS_DIR)/go-licenses-installed
 
-licenses.csv: $(GO_SOURCES) go.mod ## Track licenses in a CSV file
+licenses.csv: $(TIMESTAMPS_DIR)/go-licenses-installed go.mod ## Track licenses in a CSV file
 	@echo "Tracking licenses into file $@"
 	@echo "========================================"
-	$(GO_LICENSES) csv --include_tests $(BASE_GO_PACKAGE)/... > $@
+	GOOS=linux GOARCH=amd64 $(GO_LICENSES) csv --include_tests $(BASE_GO_PACKAGE)/... > $@
```

Comment:
out of curiosity, why do you need to explicitly set OS and ARCH?

Reply:
We want to create the licenses for the same ARCH and OS always, so they do not differ if they were computed on your Mac or in the CI.

Unfortunately, this does not really fix all the cases. Right now I run `make licenses.csv` at my computer and the CI finds differences when it repeats that check.